docs/ocis/deployment/ocis_oc10_backend.md

```diff
@@ -24,8 +26,8 @@ ocis will allow owncloud 10 users to log in and work with their files.
 
 ### Requirements
 
-* Server running Ubuntu 20.04 is public availible with a static ip address
-* Two A-records for both domains are pointing on the servers ip address
+* Server running Ubuntu 20.04 is public available with a static ip address
+* Two A-records for both domains are pointing to the servers ip address
```

Comment:
maybe explain why? in order to prepare the user by user migration? running the two instances in parallel?